source/common/http/header_utility.h

```diff
@@ -45,18 +45,37 @@ class HeaderUtility {
     Regex::CompiledMatcherPtr regex_;
     envoy::type::Int64Range range_;
     const bool invert_match_;
+
+    /**
+     * Returns true if any of the headers in HeaderMap matches this HeaderData.
+     */
+    bool matchesHeaders(const HeaderMap& headers) const override {
```

Comment:
nit: our style is to just put a single comment above a block of interface implementations, so this would just be:
```
// HeaderMatcher
bool matchesHeaders(const HeaderMap& headers) const override {
...
```

Reply:
Done.